### DISCUSSION TITLE
Rubrics of SE HW1

### DISCUSSION THREAD

**Discussion by Arun152k** [rohitgeddam/FindMyRoomie#82] [General]
Guys, I have created folders in 'develop' branch. They were a part of SE HW1 rubrics. Some are empty. Someone, please check and fill them out. Also, these are the remaining things from HW1:

1. Test folder with README
2. code/init.py
3. Data/init.py
4. Setup.py

I will create an issue for the same.

**Comment by kironjayesh** [comment]
Yes, I am making changes to CONTRIBUTING as well, to make it more extensive.
